docs/source/format/Metadata.rst

```diff
@@ -390,7 +387,58 @@ Time supports the same unit resolutions: second, millisecond, microsecond, and
 nanosecond. We represent time as the smallest integer accommodating the
 indicated unit. For second and millisecond: 32-bit, for the others 64-bit.
 
-Dictionary encoding
--------------------
+Custom Application Metadata
+---------------------------
+
+We provide a ``custom_metadata`` field at three levels to provide a
+mechanism for developers to pass application-specific metadata in
+Arrow protocol messages. This includes ``Field``, ``Schema``, and
+``Message``.
+
+The colon symbol ``:`` is to be used as a namespace separator. It can
+be used multiple times in a key.
+
+The ``ARROW`` pattern (colon included) is a reserved namespace for
```

Comment:
Did you mean `ARROW:`, as "colon included" implies?

Reply:
I'll remove the "colon included" part =) That a colon must be added stems from the namespace separator line previously